resources/sass/app.scss

```diff
@@ -262,70 +260,71 @@ dl.import-meta .download-csv {
 
 .button {
     padding: 6px 13px;
```

Comment:
Value should be:

$wikit-Button-medium-padding-vertical $wikit-Button-medium-padding-horizontal

Reply:
Done in bba763be328c5e94d3eda0be8acab4717fca42cc.